### ISSUE TITLE
Functions are not listed

### ISSUE THREAD

**Issue by Arcesilas** [tocjent/php-integrator-symbol-viewer#27]
The title should be self-explanatory :)

Functions are still alive and can be used as helpers, whether they are namespaced or not.

It could be nice to have them listed as symbols, since... they actually are symbols :)
It's possible to navigate through them, too ^^

Thanks !

**Comment by tocjent** [comment]
Hello,

that would be present in the next release, although I'm not sure when would I be able to release it, probably not until mid-April.

Thanks you your involvement!
